contracts.md

```diff
@@ -91,17 +91,17 @@ version represents the currently active contract.
 ## Distributor
 
 The distributor contract receives minted OHM from the treasury in order to
-drip-feed rewards to stakers. This means all OHM in the distributor is backed by
-DAI, because it is already being accounted for by the protocol. The reward rate
-as of time of writing is set to 5700, which translates to 0.57% of circulating
-supply, since the reward rate is defined in tens of thousands. Note that the
-reward rate determines the rebase rate and that the rebase rate determines the
-APY. Below are listed distributor contracts by version, where the latest version
-represents the currently active contract.
+drip-feed rewards to stakers. The reward rate target as of time of writing is
+set to 3300, which translates to 0.35% of total supply, since the reward rate is
```

Comment:
```suggestion
set to 3500, which translates to 0.35% of total supply, since the reward rate is
```

Reply:
You are right. I got confused. Though it is that the distributor is set to 3300 for the new staking contract. The reason is because of the old staking contract still receiving rewards. Both the old and the new staking contract add up to 0.35%. So this is what the distributor gives out in total after the target is reached.